admin/app/templates/views/terms-of-use.html

```diff
@@ -13,46 +13,393 @@
+      team has access to certain parts of the service.
+    </p>
+    <p>
+      Notify uses two-factor authentication. To sign in to Notify, you’ll need
+      to enter:
+    </p>
+    <ul class="list list-bullet">
+      <li>
+        your government email address and your Notify password
+      </li>
+      <li>
+        a text message code that Notify sends to your mobile phone
+      </li>
+    </ul>
+    <p><strong>Open source</strong></p>
+    <p>
+      In line with the Digital Service Standard, all code behind Notify is open
+      source and available on
+      <a href="https://github.com/govau/notify">our GitHub repository</a>.
+      Notify uses JSON Web Tokens, so API keys aren’t sent over the wire.
+    </p>
+    <p><strong>How we manage risks on Notify</strong></p>
+    <p>
+      Things we do to manage risks on Notify include:
+    </p>
+    <ul class="list list-bullet">
+      <li>regular privacy reviews</li>
+      <li>security impact assessments</li>
+    </ul>
+    <p><strong>Raising issues</strong></p>
+    <p>
+      Anyone can raise an issue for Notify. You can
+      <a href="mailto:notify-support@digital.gov.au">contact us</a> us to:
+    </p>
+    <ul class="list list-bullet">
+      <li>report a bug</li>
+      <li>request a feature</li>
+      <li>raise a security concern</li>
+    </ul>
+    <p>
+      Our service standard to responding to raised issues is 48 hours. We will
+      endeavour to address critical issues as soon possible, dependent on
+      resourcing restraints.
+    </p>
+    <h2 class="heading-medium" id="privacy-and-data-controls">
+      Privacy and data controls
+    </h2>
+    <p>
+      In Notify, your organisation is the data controller and the DTA the data
+      processor. Both parties are legally responsible for compliance with the
+      Privacy Act 1988 (Cth) and other relevant legislation.
+    </p>
+    <p>In Notify, data is encrypted:</p>
+    <ul class="list list-bullet">
+      <li>
+        when it passes through the service
+      </li>
+      <li>
+        when it’s stored on the service
+      </li>
+    </ul>
+    <p>
+      Any user data you upload is only held for 7 days, after which time it will
+      be deleted.
+    </p>
+    <p>Notify complies with:</p>
+    <ul class="list list-bullet">
+      <li>
+        the obligations under the Privacy Act 1988 (Cth)
+      </li>
+      <li>
+        Australian Government Agencies
+        <a
+          href="https://www.oaic.gov.au/privacy-law/australian-government-agencies-privacy-code/"
+          >Privacy Code</a
+        >
+      </li>
+    </ul>
+    <p><strong>Protect sensitive information</strong></p>
+    <p>
+      Notify makes a copy of messages you send available through the dashboard
+      for 7 days after sending.
+    </p>
+    <p>
+      Some messages that you send include sensitive information like security
+      codes or password reset links. Messages you send will likely also include
+      personal information.
```

Comment:
I feel like this should be:
`
Messages that you send may include sensitive information like security
      codes or password reset links. They may also include
      personal information.
`